velox/exec/WindowPartition.h

```diff
@@ -189,5 +226,14 @@ class WindowPartition {
   // corresponding indexes of their input arguments into this vector.
   // They will request for column vector values at the respective index.
   std::vector<exec::RowColumn> columns_;
+
+  // The partition offset of the first row in rows_.
+  vector_size_t startRow_ = 0;
+
+  // Indicates that the partial window partitioning process has been completed.
+  bool complete_ = false;
```

Comment:
Maybe move these variables to the beginning of the protected section so that they can be set in the constructor.